packages/taiko-client/driver/preconf_server/api.go

```diff
@@ -53,13 +53,13 @@ type CreateOrUpdateBlocksFromBatchResponseBody struct {
 // if the preconfirmation block groups in request are valid, it will insert the correspoinding new preconfirmation
 // blocks to the backend L2 execution engine and return a success response.
 //
-//	@Summary	Insert preconfirmation blocks by the given groups to the backend L2 execution engine, please note that
-//	            the AVS service should sort the groups and make sure all the groups are valid at first.
+//	@Description	Insert preconfirmation blocks by the given groups to the backend L2 execution engine, please note that
+//	@Description    the AVS service should sort the groups and make sure all the groups are valid at first.
 //	@Param    body body CreateOrUpdateBlocksFromBatchResponseBodyRequestBody true "preconf blocks creation request body"
 //	@Accept	  json
 //	@Produce	json
 //	@Success	200		{object} CreateOrUpdateBlocksFromBatchResponseBody
-//	@Router		/perconfBlocks [post]
+//	@Router		/preconfBlocks [post]
```

Comment:
```suggestion
//	@Router		/preconfTransactions [post]
```

Reply:
Shall we use ResetPreconfirmationBlockHeight? If the only parameter is a block ID/height, we should use height, as "head" usually means a structure.

Reply:
@davidtaikocha

Reply:
After another thought: Lets name these two functions:

- build_tentative_blocks (buildTentativeBlocks)
- remove_tentative_blocks (removeTentativeBlocks)

Reply:
> After another thought: Lets name these two functions:
> 
> * build_tentative_blocks (buildTentativeBlocks)
> * remove_tentative_blocks (removeTentativeBlocks)

Updated the method names and the URL path